src/utils/helpers/poller/__spec__.js

```diff
@@ -135,6 +134,22 @@ describe('poller', () => {
       });
     });
 
+    describe('when polling terminates and theres an onMaxRetries function', () => {
+      it('calls the onMaxRetries function', () => {
+        const onMaxRetriesSpy = jest.fn();
+        functions.onMaxRetries = onMaxRetriesSpy;
+
+        Poller({ url }, functions, { interval: 1000, retries: 2 });
+        jest.runTimersToTime(1000);
+
+        for (let i = 0; i < 2; i++) {
```

Comment:
do you want to asset above this that the function has NOT been called? having your assertion that it has been called at the bottom does not assert _when_ it was called.

Reply:
@andrewjtait - just tidying up this PR. Is your comment answered by line 146?